### PR TITLE
Clippy fixes

### PR DESCRIPTION
**Pull request by barvirm**

### CODE REVIEW

**Comment by mattfbacon** [comment]
#285

**Comment by barvirm** [comment]
> #285

I know about this PR. But it's a little too much at once. I do it in multiple PR after I finish default clippy compliance I want to change the GitHub workflow to contain check for clippy compliance and then slowly introduce other clippy restrictions.

**Comment by mattfbacon** [comment]
That seems like a good idea. Ideally I would like to make one PR for each item that I listed in the long comment on my PR.. but also I'm waiting on maintainer feedback to determine which I should actually fix and which I should leave as is

File: library/src/visualize/image.rs [hunk 1]
```diff
@@ -129,8 +129,7 @@ impl LocalName for ImageElem {
             Lang::RUSSIAN => "Рисунок",
             Lang::SLOVENIAN => "Slika",
             Lang::SPANISH => "Figura",
-            Lang::UKRAINIAN => "Рисунок",
```
Comment (laurmaedje) [inline library/src/visualize/image.rs:132]: why remove Ukrainian?

File: library/src/layout/table.rs [hunk 2]
```diff
@@ -298,7 +298,7 @@ impl LocalName for TableElem {
             Lang::SLOVENIAN => "Tabela",
             Lang::SPANISH => "Tabla",
             Lang::UKRAINIAN => "Таблиця",
-            Lang::ENGLISH | _ => "Table",
```
Comment (laurmaedje) [inline library/src/layout/table.rs:301]: I think it's clearer with the explicit `Lang::ENGLISH`

Comment (barvirm) [reply library/src/layout/table.rs:304]: We can disable `clippy::wildcard_in_or_patterns` or split it into two lines:
```
Lang::ENGLISH => "Table",
_ => "Table"
```

Comment (laurmaedje) [reply library/src/layout/table.rs:304]: I would prefer disabling this clippy lint globally as whenever I do this, it is for a reason. Duplicating the string isn't that nice.

Comment (barvirm) [reply library/src/layout/table.rs]: @laurmaedje Done

**Pull request merged**